bouncer/shared/utils/chainflip_io.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  ChainflipExtrinsicSubmitter,
+  createStateChainKeypair,
+  extractExtrinsicResult,
+  lpMutex,
+} from 'shared/utils';
+import { z } from 'zod';
+// eslint-disable-next-line no-restricted-imports
+import type { KeyringPair } from '@polkadot/keyring/types';
+import { DisposableApiPromise, getChainflipApi } from './substrate';
+import { ChooseSingleEvent, EventDescription, EventName, findOneEventOfMany } from './indexer';
+import { Logger } from './logger';
+
+export class ChainflipIO<Requirements> {
+  /// The last block height at which either an input or an output operation happened,
```

Comment:
Looks like you've used rust-style doc comments everywhere 😅 

In order for the IDE to display it in the tooltip it needs to be formatted like `/** BLAH */`